sprz.cls

```diff
@@ -15,13 +15,13 @@
 \RequirePackage[utf8]{inputenc}
 \RequirePackage{polski} % Włączenie obsługi polskich liter
 \RequirePackage{tabularx}
-\RequirePackage[backend=bibtex,style=numeric,sorting=none]{biblatex}
+\RequirePackage[backend=biber,style=numeric,sorting=none]{biblatex}
 \RequirePackage{url}
 \RequirePackage{graphicx}
 \RequirePackage[pscoord]{eso-pic}% The zero point of the coordinate systemis the lower left corner of the page (the default).
 \RequirePackage{multirow}
 \RequirePackage{graphicx}
-
+\RequirePackage{csquotes}
```

Comment:
Why introduce additional package that is not necessary?

Reply:
To remove warning from build logs. I went the other way and removed `biblatex` from template.